pipelines/templates/1es-publish-task.yml

```diff
@@ -0,0 +1,22 @@
+parameters:
+  - name: path
+    type: string
+
+  - name: artifactName
+    type: string
+
+  - name: displayName
+    type: string
+    default: 'Publish artifact'
```

Comment:
Nit: quotes are unnecessary here. Only if it starts with special characters like a number, `*`, and a few others I don't recall. Doesn't hurt, though. Just makes you look paranoid. ;)

Reply:
...i.e. values can totally have spaces in them.